tasks/lzards-backup/src/index.ts

```diff
@@ -42,7 +42,7 @@ import {
 
 const log = new Logger({ sender: '@cumulus/lzards-backup' });
 
-const CREDS_EXPIRY_SECONDS = 1000;
+const CREDS_EXPIRY_SECONDS = 3600;
```

Comment:
If the cred expires before s3 presigned url, we will get
<Code>ExpiredToken</Code>
 <Message>The provided token has expired.</Message>

Reply:
```suggestion
const S3_LINK_EXPIRY_SECONDS_DEFAULT = 3600;
const CREDS_EXPIRY_SECONDS = S3_LINK_EXPIRY_SECOND_DEFAULT;
```

Would it be reasonable to set `CREDS_EXPIRY_SECONDS` using `S3_LINK_EXPIRY_SECOND_DEFAULT` if it is dependent on it?